tests/hpc/conman.pm

```diff
@@ -0,0 +1,87 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: HPC_Module: Add test for conman  package
+#
+#    https://fate.suse.com/321724
+#
+#    This tests the conman package from the HPC module
+#
+# Maintainer: Matthias Griessmeier <mgriessmeier@suse.com>
+
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+use susedistribution;
+
+sub run() {
+    select_console 'root-console';
+
+    # install conman
+    pkcon_quit();
+    zypper_call('in conman');
+
+    # add serial console to conman.conf
+    assert_script_run("echo 'CONSOLE name=\"serial1\" dev=\"/dev/$serialdev\" seropts=\"115200\"' >> /etc/conman.conf");
+    script_run("cat /etc/conman.conf");
+
+    # enable and start conmand
+    assert_script_run('systemctl enable conman.service');
```

Comment:
we doing same things for slurm and other munge and I am suspecting that we will use same pattern for several more. So it would be cool if you will move this pattern (enable service / start service / check status) to some function which we can call